### ISSUE TITLE
Is it possible to use this project directly for a code example for instrument recognition?

### ISSUE THREAD

**Issue by 980202006** [kkoutini/PaSST#15]
Can I output the labels directly with the pretrained model or do I need to do fine-tuning for openmic-2018

**Comment by kkoutini** [comment]
you can use the fine-tuned model on openmic [here](https://github.com/kkoutini/PaSST/releases/tag/v0.0.5) [openmic-passt-s-f128-10sec-p16-s10-ap.85.pt](https://github.com/kkoutini/PaSST/releases/download/v0.0.5/openmic-passt-s-f128-10sec-p16-s10-ap.85.pt)

```shell
!pip install git+https://github.com/kkoutini/passt_hear21.git
```
```python
import torch

from hear21passt.base import load_model
model = load_model(mode="logits").cuda()
state = torch.load("/path/to/openmic-passt-s-f128-10sec-p16-s10-ap.85.pt")
model.net.load_state_dict(state)
logits = model(wave_signal)

```

**Comment by 980202006** [comment]
What is the order of the label set of logits？

**Comment by kkoutini** [comment]
the labels are in the same order as in the "openmic-2018.npz" file in the dataset [here](https://github.com/cosmir/openmic-2018/blob/master/examples/modeling-baseline.ipynb). I think it's the chronological order of the classes.

**Comment by 980202006** [comment]
Thank you!
